addons/portal/views/portal_templates.xml

```diff
@@ -334,7 +334,7 @@
 
     <template id="portal_contact" name="Contact">
         <div class="o_portal_contact_details mb-5">
-            <h4><t t-if="title" t-esc="title"/><t t-else="">Your contact</t></h4>
```

Comment:
> @qsm-odoo You can review this PR heavy_check_mark . This PR is enough to solve the problem but to be clean, it is necessary to merge https://github.com/odoo/odoo/pull/108705. I let you decide if we merge the community PR in 14.0 or if I retarget the community PR on master.

@Guillaume-gdi You should know by now that you cannot alter the structure of the XML in stable unless critical and really no other way (in which case I should see an explanation here somewhere). So if the enterprise PR is enough as you said, this could be for master. Maybe this community PR could be adapted with a comment above the template to explain the "title" variable is actually buggy and to adapt in master etc.

However, are you sure the enterprise PR is enough? If you simply remove the "Salesperson" title given to the rendering, won't this use the default page "title" and thus never display "Your contact"? :thinking: If not, why? :thinking:

Reply:
I confirm that the enterprise PR is sufficient to fix the problem. "Your contact" will not be replaced by the page title. The fix works because of the way the template engine organizes the scope of variables with t-calls and inherit. The `/my` route render the template `portal_my_home`. To summarize our case, we have this:
```
<template id="portal_my_home" name="My Portal">
    <t t-call="portal.portal_layout">
        ...
    </t>
</template>

<template id="portal_layout" name="Portal Layout">
    <t t-call="portal.frontend_layout">
        ...
        <t t-call="portal.portal_contact"/>
        ...
    </t>
</template>

<template id="portal_contact" name="Contact">
    ...
    <h4><t t-if="title" t-esc="title"/><t t-else="">Your contact</t></h4>
    ...
</template>

<template id="frontend_layout" name="Main Frontend Layout" inherit_id="web.frontend_layout">
    ...
</template>

<template id="web.frontend_layout" name="Frontend Layout" inherit_id="web.layout" primary="True">
    ...
</template>

<template id="web.layout" name="Web layout">
    <title t-esc="title or 'Odoo'"/>
</template>
```
and when website is installed we have this additional template:
```
<template id="layout" name="Main layout" inherit_id="portal.frontend_layout">
    ...
    <t t-set="title" t-value="default_title"></t> <!-- My portal | My website -->
    ...
</template>
```
The scope of the **title** variable setted by the website (or the web) module is only available in `<template id="layout"`, `<template id="web.layout"`, `<template id="web.frontend_layout"` and `<template id="frontend_layout"` but it will not be for `<template id="portal_contact"`.